### ISSUE TITLE
Intellisense, linting failing on Mac Catalina out of the box

### ISSUE THREAD

**Issue by jonahpearl** [microsoft/vscode#115228]
- VSCode Version: 1.52.1
- OS Version: 10.15.7 (19H114)

Steps to Reproduce:

1. Uninstall VS Code as described [here](https://medium.com/@jimkang/complete-uninstall-remove-vscode-mac-5e48bef3bdec)
2. Download VS Code for Mac (stable build) from the website, run installer, move to Applications folder.
3. Open and launch the "Interactive Playground" tutorial.
4. Errors have double red underlines, Intellisense example does not connect to any API (see image).
5. Install Python extension and start coding in Python -- same issues. Linters don't work in real time, Intellisense doesn't work at all.

Other things like multi-cursor editing, refactoring, all work fine. I don't often submit bug reports here -- please tell me if you need more info. Thank you!

![image](https://user-images.githubusercontent.com/68478436/106004417-315a9600-6081-11eb-84a5-2eb2c3f10c8b.png)

Does this issue occur when all extensions are disabled?: **Yes**/No

**Comment by mjbvz** [comment]
Please open a separate issue against the Python extension for python related problems

**Comment by mjbvz** [comment]
Here's what I see for the interactive playground in VS Code insiders:

<img width="1349" alt="Screen Shot 2021-01-27 at 10 48 20 AM" src="https://user-images.githubusercontent.com/12821956/106038835-450bf980-608d-11eb-8802-6fbe6217b2b3.png">

The Intellisense depends on [automatic type acquisition](https://code.visualstudio.com/docs/nodejs/working-with-javascript#_typings-and-automatic-type-acquisition). This means that types need to be downloaded before we can provide intellisense. Did you see a message in the status bar about `Fetching data for better TypeScript IntelliSense`

**Comment by jonahpearl** [comment]
No -- I just see one for a few seconds that says "Initializing JS/TS language features." Then status bar is empty.

**Comment by mjbvz** [comment]
Do you have npm installed on your system?

**Comment by jonahpearl** [comment]
Was about to say — no, I don't! Didn't know it was required for VS Code. Are there other dependencies listed somewhere? Installing now and we'll see.

**Comment by mjbvz** [comment]
Yes if you are working with a node module such as express, then you do need to install it.

@chrmarti Should we update the example to either not depend on npm or note that you need npm installed?

@jonahpearl You see the double underline because you are using a high contrast theme. This indicates that there is an error in the code (there needs to be something after the `.`)

**Comment by jonahpearl** [comment]
Aha, I see. It was set to that high contrast theme by default for some reason...changing it to Dark+ gives the expected appearance. Thank you, will file a separate issue for the Python extension.

**Comment by chrmarti** [comment]
@mjbvz Makes sense. Changing it to:
```js
const canvas = document.querySelector('canvas');
const context = canvas.getContext('2d');

context.strokeStyle = 'blue';
context.
```
